ccbc-domain/src/main/java/io/coin/ccbc/domain/CoinHistory.java

```diff
@@ -35,7 +35,19 @@
   @Column(name = "balance", nullable = false, precision = 78)
   private BigInteger balance;
 
-  public static CoinHistory collected(Address wallet, Coin coin, BigInteger balance) {
-    return new CoinHistory(wallet, coin, balance);
+  private CoinHistory(Address wallet, Coin coin, BigInteger balance, LocalDateTime time) {
+    super(time);
+    this.walletAddress = wallet;
+    this.coin = coin;
+    this.balance = balance;
+  }
+
+  public static CoinHistory collected(
+      Address wallet,
+      Coin coin,
+      BigInteger balance,
+      LocalDateTime time
```

Comment:
밖에서 쓸 때 어떤 시간이 들어가는지 모를 것 같아서 더 구체적인 이름이면 좋을 것 같아요!
```suggestion
      LocalDateTime collectedAt
```